src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -2659,8 +2659,7 @@ public TreePath getTreePath(Element e) {
      * A cache of doc comment info objects for elements.
      * The entries may come from the AST and DocCommentParser, or may be autromatically
      * generated comments for mandated elements and JavaFX properties.
-     *
-     * @see CommentUtils.dcInfoMap
+     ** @see CommentUtils#dcInfoMap
```

Comment:
I think you unintentionally removed a line break while fixing the reference.

Reply:
Oops